src/components/Dialog.js

```diff
@@ -1,8 +1,14 @@
 import classnames from 'classnames';
 import { useEffect, useLayoutEffect, useRef, useState } from 'preact/hooks';
 
-import { IconButton, LabeledButton } from '../';
-import { SvgIcon } from './SvgIcon';
+import { IconButton, LabeledButton } from './buttons';
```

Comment:
We decided recently that it's better if components import each other directly versus going through the package entry point.

Reply:
Good to know, thanks.